webapp/src/containers/components/tables/existingConfigTable.tsx

```diff
@@ -32,164 +36,476 @@ type Props = {
-                                <td>{'Standup'}</td>
-                                <td>{'4'}</td>
-                                <td className='ellipsis'>
-                                    {'Hello to standup group sdhfvk.'}
-                                </td>
-                                <td>
-                                    {'True'}
-                                </td>
-                                <td>
-                                    <div className='options'>
-                                        <ButtonGroup aria-label='Basic example'>
+            {editVisible &&
+                <ConfigModal
+                    visible={editVisible}
+                    setVis={setEditVisible}
+                    config={myConfig}
+                />
+            }
+            {addVisible &&
+                <ConfigModal
+                    visible={addVisible}
+                    setVis={setAddVisible}
+                    config={null}
+                />
+            }
+            <div className='name'>
+                {label}
+            </div>
+            <div>
+                <Table
+                    striped={true}
+                    className='existing-config-table'
+                >
+                    <thead>
+                        <tr>
+                            <th className='teamName'>{'Team Name'}</th>
+                            <th className='delay'>{'Delay (in sec)'}</th>
+                            <th className='message'>{'Message'}</th>
+                            <th className='includeGuests'>{'Include Guests'}</th>
+                            <th className='option'>{'Options'}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td className='teamName'>{'Standup'}</td>
+                            <td className='delay'>{'4'}</td>
+                            <td className='message'>
+                                {'Hello to standup group sdhfvk.'}
+                            </td>
+                            <td className='includeGuests'>
+                                {'True'}
+                            </td>
+                            <td className='option'>
+                                <div>
+                                    <ButtonGroup aria-label='Basic example'>
+                                        <OverlayTrigger
+                                            placement='top'
+                                            overlay={<Tooltip>{'View actions'}</Tooltip>}
+                                        >
                                             <Button
-                                                variant='primary'
                                                 onClick={handleView}
                                             >
-                                                {'View'}
+                                                <svg
```

Comment:
Create a separate file for these svgs and import them here.

Reply:
Will fix this in the next PR